### PR TITLE
Adds more holy weapons

### PR DESCRIPTION
**Pull request by jknpj**
nurods2

The chaos blade is a port of TG's talking blade, you can click on them as ghost or the owner can active them to have a ghost live inside of it. Right now it uses the borer role as preference to auto-assign. **It can't be selected as a skin.** You can find them in temple and war digsites if you're lucky.

Everything else is just fluffy.

:cl:
 * rscadd: Adds the staff of nullmancy, heavenly chain, honk rod, french rod, blessed cane and septerion morningstar as holy weapon skins for the null rod.
 * rscadd: Adds the chaos blade, a talking "holy" weapon, as a rare drop to temple and war digsites.

### CODE REVIEW

File: code/game/objects/items/weapons/null_rod.dm [hunk 1]
```diff
@@ -140,15 +141,126 @@
...
+	icon_state = "armblade"
+	item_state = "armblade"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	hitsound = "sound/weapons/bloodyslice.ogg"
+	cant_drop = TRUE
+
+/obj/item/weapon/nullrod/armblade/dropped()
+	var/obj/item/weapon/nullrod/N = new /obj/item/weapon/nullrod
+	qdel(src)
+	visible_message("<span class='notice'>\The [name] loses [ticker.Bible_deity_name]'s power and changes back into \a [N.name]</span>")
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/null_rod.dm]: Wouldn't it be better to have it go for the chaplains religious deity?

Comment (jknpj) [reply code/game/objects/items/weapons/null_rod.dm]: Wait, what. oh, right. we have multiple religions now

**Comment by FudgePucker** [comment]
>Chaos blade

Oh boy, I can't wait to show waifus my HYPER WEAPON and get banned for ERP.

**Comment by AutismSpider** [comment]
@FudgePucker do it as a ling

**Comment by Lokiamis** [comment]
It is excruciatingly stupid for a space station chaplain to have a morning star or an arm that’s a sword

**Comment by FudgePucker** [comment]
Where do you think we are?

**Comment by jknpj** [comment]
@Lokiamis 
> Lucifer is sometimes translated from Latin as "morning star"
> Jesus is self-described as "the bright Morning Star" in the Christian Bible
> John the Baptist is called a "bright morning star" in Eastern Orthodox Church hymnnology
> Mary the mother of Jesus, called "morning star" in the Litany of Loreto

https://github.com/vgstation-coders/vgstation13/blob/9c9925a96d23963750a14ce64566f1ff5953ede2/code/datums/religions.dm#L734-L740

**Comment by DamianX** [comment]
I thought it was agreed alternative null rod skins shouldn't have extra functionality?

**Comment by SonixApache** [comment]
>It is excruciatingly stupid for a space station

ss13

**Comment by Anglosphere** [comment]
>Chaplain getting more weapons 

Disgusting tbh. 

Religion is about love, and getting your followers to kill people for you.

**Comment by jknpj** [comment]
@DamianX still no status change, the chaos blade can only talk and the arm blade has cant_drop because it is supposed to fuse with the chaplain's arm.

The original PR also introduced size/slot changes and a few other non-stat related changes for the skins.

**Comment by Blithering** [comment]
While the ghost sword thing is neat, I think it should be separate from the null rod, and definitely not tied to one specific skin.

**Comment by ultraplen** [comment]
Is the chain icon just ganked from the chain of command and (or whatever the caps whip is called)? This PR seems like meme bloat to me, but it's not objectively bad.

**Comment by Jubisloviu** [comment]
dunno but the chaos blade and changeling arm looks like shitty ideas to be honest
i can see the novelty of the baguette, cane and maybe that thing that looks like a stunbaton with a honking horn or the morning star.
chaos blade would work better as a rare xenoarch artifact
and the armblade is just for "i'm just pretending to look like an antag, pls don't kill me sillybois"

**Comment by Pantrist** [comment]
MEMEZ

**Comment by NigglyWiggly** [comment]
For what purpose
No but really why does the chaplain need more weapons, let alone any weapons in general that don’t serve a purps like the null rod?

Also a fucking “Chaos Blade” with a spirit trapped inside of it sounds really fucking silly

**Review by PJB3005 — APPROVED** [review]

**Comment by Blithering** [comment]
@NigglyWiggly they serve the same purpose, they're just selectable skins for the null rod to go along with certain gimmick chaplains.

**Comment by jknpj** [comment]
@NigglyWiggly they are literally just skins for the null rod, it's not even MORE weapons as they can only select the skin ONCE per null rod. They are here to help chaplains use their nullrod while doing their gimmicks (Clown religion chaplain with his honkrod)

**Comment by NigglyWiggly** [comment]
Ah, thank fuck.

Still not a fan of the _**CHAOS BLADE**_ though

**Pull request merged**